.travis.yml

```diff
@@ -4,3 +4,8 @@ before_script:
   - mv ./sample-environment.yaml ./private/environment.yaml
 node_js:
   - "node"
+install:
+  - npm install
+script:
+  # - npm run-script lint
```

Comment:
Does this line need to be uncommented in order for Travis to run the lint check? I don't see any lint checks added when I click on the Travis CI link below (but maybe I'm just not finding it)

Reply:
It has to be uncommented for travis to run the linter, It is commented out since it would fail because of all the linter errors in the code. We should uncomment it once we fix them (thats why I opened #60)

If you wanna try it out, push to this branch with the line uncommented and you should see the linter errors in the travis build report

Reply:
Ah, got it. Thanks for clarifying!